com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -102,13 +101,10 @@ public Response uploadMtaFile(HttpServletRequest request, SecurityContext securi
                 continue; // ignore simple (non-file) form fields
             }
 
-            InputStream in = null;
-            try {
-                in = item.openStream();
+            try (InputStream in = item.openStream()) {
                 FileEntry entry = fileService.addFile(spaceGuid, item.getName(), getConfiguration().getFileUploadProcessor(), in);
                 uploadedFiles.add(entry);
-            } finally {
-                IOUtils.closeQuietly(in);
+            } catch (IOException ignored) {
```

Comment:
Leaving an exception with a name 'ignored' only makes Lint ignore the warning that this block is empty, but it does not make it more understandable why it actually is left empty. Stick to name 'e', and add a comment explaining why this block is empty. This is also the standard recommendation from sonar.

Reply:
If an exception occurred in the previous state of the code, it was ignored. That's why this is left like that. I will change the name to 'e' and add //Intentionally ignored.

Reply:
I understand that. Try to make it more descriptive, for example "// Close quietly", makes more sense to me when I see it than "// Intentionally ignored". The second is already apparent.